app/src/main/res/values-night/color_palette.xml

```diff
@@ -200,4 +202,29 @@
   <color name="color_palette_onboarding_4_status_bar_color">@color/color_def_onboarding_4_status_bar</color>
   <color name="color_palette_coming_soon_rect_background_end_color">@color/color_def_grape_violet</color>
   <color name="color_palette_coming_soon_rect_background_start_color">@color/color_def_bright_violet</color>
+  <!-- AVATAR BACKGROUND COLORS -->
```

Comment:
Do these need to be defined here if they're already defined in -day? I thought we only needed -night versions if they are different.

Ditto for all other colors in this XML file (including ones already checked in).

Reply:
Done.